h/resources.py

```diff
@@ -140,6 +140,42 @@ def __getitem__(self, pubid):
         return organization.logo
 
 
+class GroupResource(object):
```

Comment:
`GroupResource` and `OrganizationResource` class naming here follow the pattern established by `AnnotationResource`: `*Factory` classes retrieve models from the DB, while `*Resource` classes operate on already-extant model objects. This works well in the applied case as the list of `Group` models have already been retrieved.